src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="mt-10" id="dissolution-firm-form">
```

Comment:
Our convention is that components should have an ID and it should match the filename (and component name).